Updated to Swift 2.1.1 & Improved UI

avazquezpr opened the pull request:
## Update

App is now displayed perfectly across all iPhone and iPad screen sizes. 
Updated code to Swift 2.1.1
## Screenshots

<img width="626" alt="screen shot 2015-12-25 at 4 56 42 pm" src="https://cloud.githubusercontent.com/assets/3937516/12004343/871a5044-ab28-11e5-874b-a4aa69c99163.png">

avazquezpr committed:
Updated to latest version of Swift.

Updated to the latest version of Swift and made the demo app fit all
iPhone screens.

avazquezpr committed:
Swift ver.2.1.1

avazquezpr committed:
Small UI change.

avazquezpr committed:
Small changes to README.md & Added .gitignore

avazquezpr committed:
Ignores xcuserdata/

honghaoz commented:
@avazquezpr Great work! There are some conflicts need to be resolved. Can you fix it? Thanks for your PR!

avazquezpr committed:
Solved 3 issues.

Changed myCellNib from variable to constant. Variable i is not needed
in the for statement that calls addNewOne() method and changed the cell
in the collection view to be a constant.

avazquezpr committed:
Sign identity change.

avazquezpr commented:
Sure, no problem! What are the conflicts?

honghaoz commented:
<img width="795" alt="screen shot 2016-01-06 at 1 34 28 pm" src="https://cloud.githubusercontent.com/assets/3926785/12150881/3e958e82-b47a-11e5-9f1a-01e52688e138.png">
Did you pull from origin master? (My repo master branch). Since i cannot merge it

avazquezpr commented:
Yes, i pulled from honghaoz/master. It must be because there are new commits that are not in my pull request's repo. Will work on that now :+1:

honghaoz commented on DynamicCollectionViewCellWithAutoLayout-Demo/ViewController.swift:
```diff
@@ -143,16 +147,5 @@ class ViewController: UIViewController, UICollectionViewDataSource, UICollection
-    // iOS8
-    override func viewWillTransitionToSize(size: CGSize, withTransitionCoordinator coordinator: UIViewControllerTransitionCoordinator) {
-        collectionView.collectionViewLayout.invalidateLayout()
-    }
```
why those methods are removed?

avazquezpr replied:
My mistake. Fixed.

honghaoz commented on README.md:
```diff
@@ -2,12 +2,12 @@
 ## Updated
 
-For off screen cells, there's a new dynamic collection view which will handle it gracefully. Check it out!
-[ZHDynamicCollectionView](https://github.com/honghaoz/ZHDynamicCollectionView)
```
This shouldn't be removed.

honghaoz commented:
everything else looks good :+1:

avazquezpr committed:
Restored files.

Added missing methods to ViewController and changed the Updated part of
the README.md

avazquezpr committed:
Merge remote-tracking branch 'honghaoz/master'

# Conflicts:
#	README.md

avazquezpr committed:
Fixed Merge Conflicts.

Fixed conflicts that prevented merge. Updated the README.md file.

avazquezpr commented:
Fixed the conflicts :+1: Let me know if there is anything else i can help with!

honghaoz commented:
:+1: Thanks @avazquezpr !
